scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl

```diff
@@ -85,178 +91,215 @@ workflow GvsCreateVATfromVDS {
     String effective_vat_version = if (defined(vat_version) && select_first([vat_version]) != "v1") then "_" + select_first([vat_version]) else ""
     String vat_table_name = filter_set_name + "_vat" + effective_vat_version
 
-    if (!defined(split_intervals_scatter_count)) {
-        call Utils.GetBQTableLastModifiedDatetime as SampleDateTime {
+    String output_path_without_a_trailing_slash = sub(output_path, "/$", "")
+    if (output_path == output_path_without_a_trailing_slash) {
+        call Utils.TerminateWorkflow as OutputPathMustEndWithASlash {
             input:
-                project_id = project_id,
-                fq_table = "~{project_id}.~{dataset_name}.sample_info",
-                cloud_sdk_docker = effective_cloud_sdk_docker,
+                message = "Error: 'output_path' MUST end with a trailing '/'",
+                basic_docker = effective_basic_docker,
         }
+    }
 
-        call Utils.GetNumSamplesLoaded {
+    if (!defined(sites_only_vcf) && ((!defined(vds_path) || !defined(hail_generate_sites_only_script_path)))) {
```

Comment:
Maybe treat the two cases separately? If `sites_only_vcf` then those other two params really shouldn't be?

Reply:
Yeah. I wondered about it. I kind of figured there's no harm if a user leaves an unneeded parameter defined, but it could cause confusion.

Reply:
Okay - did that